components/uniset/src/builder.rs

```diff
@@ -381,6 +381,20 @@ impl UnicodeSetBuilder {
     pub fn complement_set(&mut self, set: &UnicodeSet) {
         self.complement_list(set.as_inversion_list());
     }
+
+    /// Returns whether the build is empty.
+    ///
+    /// # Example:
```

Comment:
We don't have the trailing colon in other docstrings: `/// # Example`

Reply:
Done.

Reply:
It looks like all the docstrings in the uniset component have a trailing colon, so I removed them.